test/Microsoft.SqlTools.ServiceLayer.UnitTests/Connection/ConnectionServiceTests.cs

```diff
@@ -647,6 +648,31 @@ public void BuildConnectionStringWithInvalidOptions(string propertyName, object
                 }
         };
 
+        private static readonly object[] EncryptionCombinations =
+        {
+            new object[] {true, true, SqlConnectionEncryptOption.Strict },
+            new object[] {true, false, SqlConnectionEncryptOption.Mandatory },
+            new object[] {false, true, SqlConnectionEncryptOption.Optional },
+            new object[] {false, false, SqlConnectionEncryptOption.Optional },
+        };
+        
+        /// <summary>
+        /// Verify that Strict Encryption parameters can be built into a connection string for connecting.
+        /// </summary>
+        [Test, TestCaseSource(nameof(EncryptionCombinations))]
```

Comment:
You can just use TestCase here to inline these - shouldn't need to use a TestCaseSource